storage/src/test/java/com/tencent/rss/storage/common/LocalStorageTest.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available. 
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved. 
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.storage.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import com.google.common.collect.Lists;
+import com.tencent.rss.common.util.RssUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.roaringbitmap.RoaringBitmap;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+
+public class LocalStorageTest {
+
+  @ClassRule
+  public static final TemporaryFolder tmpDir = new TemporaryFolder();
+  private static File testBaseDir;
+
+  @BeforeClass
+  public static void setUp() throws IOException  {
+    testBaseDir = tmpDir.newFolder("test");
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    tmpDir.delete();
+  }
+
+  @Test
+  public void canWriteTest() {
```

Comment:
`public void canWriteTest() throw Exception`, no try-catch block in method

Reply:
OK